pkg/handler/handlers.go

```diff
@@ -112,7 +113,7 @@ func SaveDeveloperProfileHandler(svr server.Server) http.HandlerFunc {
 			svr.JSON(w, http.StatusBadRequest, "too many skills")
 			return
 		}
-		existingDev, err := database.DeveloperProfileByEmail(svr.Conn, req.Email)
+		existingDev, err := svr.DeveloperRepo.DeveloperProfileByEmail(req.Email)
```

Comment:
once you pass the repository into the handler this will become `devRepo.DeveloperProfileByEmail(req.Email)` so not big change :)